content/applications/sales/sales/ebay_connector/deletion_notifications.rst

```diff
@@ -0,0 +1,71 @@
+===============================================
+eBay Marketplace account deletion notifications
+===============================================
+
+Since September 2021, **eBay requires supporting customer account deletion/closure notifications**.
+As such, when eBay receives an account request for deletion, all eBay partners must confirm the
+reception of the request and take further action if necessary.
+
+Odoo has a notification endpoint to receive those notifications and handle the first set of actions
+to anonymize the account details in Contact and remove the customer's access to the portal.
+
+.. warning::
+   Make sure to correctly set up your subscription to the **marketplace account deletion
+   notifications**, as eBay may temporarily disable the related eBay account until the subscription
+   is completed.
```

Comment:
We should make a relative reference to the related step below to let the user know that this part is covered.